MetaMask/metamask-extension
# Feat refresh accounts

## Explanation

Auto detect accounts when press Refresh button
New design:
https://www.figma.com/file/33geacOStnlnJeBZhquDn7/Refresh-Accounts?node-id=5%3A3143



## More information

Fixes #13276



## Screenshots/Screencaps



### Before
<img width="985" alt="image" src="https://user-images.githubusercontent.com/97883527/158561305-d6f830d9-0e99-43f8-90e4-316332e93f25.png">



### After
<img width="741" alt="image" src="https://user-images.githubusercontent.com/97883527/158561058-c1271e47-759a-4f83-94a1-0cc1f5b51ba2.png">



## Manual testing steps

- Go to MetaMask extension
- Click on profile picture, then refresh accounts
- Notice list of accounts being refreshed



Auto detect accounts when switching networks 

https://user-images.githubusercontent.com/97883527/159643972-fa9dcb14-1be5-49a2-9685-a349d2f37c1c.mov

Auto detect accounts when login to MetaMask

https://user-images.githubusercontent.com/97883527/159644568-5f4e353c-628e-4b04-9494-5faddeae34cb.mov

Add a new account with the funds on different device, and press refresh button to detect that account

https://user-images.githubusercontent.com/97883527/159644912-6478954b-381e-4d9e-b632-57cf3a8333dc.mov


New design: 
https://www.figma.com/file/33geacOStnlnJeBZhquDn7/Refresh-Accounts?node-id=5%3A3143
